app/presenters/summary/html_presenter.rb

```diff
@@ -72,27 +88,36 @@ def initialize(crime_application:)
     end
 
     def sections
-      SECTIONS.fetch(application_type.to_sym).map do |section|
-        Sections.const_get(section.to_s.camelize).new(crime_application)
-      end.select(&:show?)
+      if appeal_no_changes?
+        build_sections(:appeal_with_no_changes)
+      else
+        build_sections(application_type.to_sym)
+      end
     end
 
-    def capital_sections
-      SECTIONS.fetch(:capital).map do |section|
-        Sections.const_get(section.to_s.camelize).new(crime_application)
-      end.select(&:show?)
+    def income_sections
+      build_sections(:income)
     end
 
     def outgoings_sections
-      SECTIONS.fetch(:outgoings).map do |section|
-        Sections.const_get(section.to_s.camelize).new(crime_application)
-      end.select(&:show?)
+      build_sections(:outgoings)
     end
 
-    def income_sections
-      SECTIONS.fetch(:income).map do |section|
+    def capital_sections
+      build_sections(:capital)
+    end
+
+    private
+
+    def build_sections(relevant_sections)
+      SECTIONS.fetch(relevant_sections).map do |section|
         Sections.const_get(section.to_s.camelize).new(crime_application)
       end.select(&:show?)
     end
+
+    def appeal_no_changes?
```

Comment:
Could this live in "TypeOfApplication"

Reply:
That would be the ideal place for it to be, it just didn't conceptually feel like a different kind of application - does it fit with your model of being a different kind of application @timpeat 

And if so where does that stop (i.e. full vs initial means assessment for example)